Objects/typeobject.c

```diff
@@ -3012,6 +3012,12 @@ PyType_FromModuleAndSpec(PyObject *module, PyType_Spec *spec, PyObject *bases)
         else if (slot->slot == Py_tp_doc) {
             /* For the docstring slot, which usually points to a static string
                literal, we need to make a copy */
+
+            /* bpo-41832: PyType_FromModuleAndSpec() can accept tp_doc=NULL. */
```

Comment:
IMO this comment is useless, please remove it.

Reply:
done.